src/components/SourceTabs.js

```diff
@@ -124,12 +146,25 @@ const SourceTabs = React.createClass({
       click: () => showSource(tab)
     };
 
+    const copySourceUrl = {
+      id: "node-menu-close-tabs-to-right",
+      label: "Copy Link Address",
+      accesskey: "X",
+      disabled: false,
+      click: () => {
+        showSource(tab);
+        this.copyToTheClipboard();
```

Comment:
ahh, interesting...

I suppose we could get the `source` above https://github.com/arthur801031/debugger.html/blob/46b268f751aa73a55a04c3b5394ea146a26f6957/src/components/SourceTabs.js#L104.

```js
const source = tabs.find(t => t.get("id") == tab)
```

or something like that...